packages/devtools_app/lib/src/config_specific/flutter/drag_and_drop/_drag_and_drop_desktop.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '_drag_and_drop_base.dart';
+
+// TODO(kenz): implement once Desktop support is available. See
+// https://github.com/flutter/flutter/issues/30719.
+
+class DragAndDrop extends DragAndDropBase<PointerEvent> {
+  const DragAndDrop({
+    @required void Function(PointerEvent event) onDrop,
+    @required Widget child,
+  }) : super(onDrop: onDrop, child: child);
+
+  @override
+  _DragAndDropState createState() => _DragAndDropState();
+}
+
+class _DragAndDropState extends DragAndDropBaseState<PointerEvent> {
+  @override
+  void dispose() {
```

Comment:
cleaup all no-op overrides that are just calling the base class here and in drag_and_drop_web.